CDF Outreach Committee Charter.md

```diff
@@ -0,0 +1,115 @@
+
+# CDF Outreach Committee Charter
+
+
+## Mission Statement
+
+The Outreach Committee (OC) is the place where the community comes together, including the projects, the members, and the ambassadors with the goal to identify opportunities to help market and communicate Continuous Delivery Foundation's (CDF) vision, projects, Special Interest Groups (SIG) and Working Groups (WG) (new initiatives, new releases, need for contributors, etc.) within the CDF community and to the broader technical community in general. The OC Chair acts as the communication link between the board and the rest of the community.
+
+
+## Members
+
+Members of the OC should consist of one representative from each member company at the Premier, General  and End User membership levels as well as one representative for all adopted CDF projects and the Ambassador Program Chair. This will make up the voting board to be led with direction from the OC Chair. Additional project and community members may attend OC meetings in a non-voting capacity. Members will be expected to contribute to initiatives aimed at growing the CDF community and supporting CDF projects.
+
+**Voting Members**
+
+
+
+* Member Company Representatives
+* Project Representatives
+* Ambassador Program Chair
+
+**Non Voting**
+
+
+
+* Observers
+* Governing Board
+* Ambassadors
+
+
+## Communication
```

Comment:
what about asynchronous communication? Do we need to include info about Slack for example?